Makefile

```diff
@@ -534,11 +535,12 @@ validate-runtime-image: bin/kubectl
 		fi; \
 		if [ $$cmd == "python3" ]; then \
 			echo "=> Checking notebook execution..." ; \
-			$(KUBECTL_BIN) exec runtime-pod -- /bin/sh -c "python3 -m pip install -r /opt/app-root/elyra/requirements-elyra.txt && \
+			$(KUBECTL_BIN) exec runtime-pod -- /bin/sh -c "curl https://raw.githubusercontent.com/opendatahub-io/elyra/refs/heads/main/etc/generic/requirements-elyra.txt --output req.txt && \ 
```

Comment:
Tried this in my PR, as I said and would, and I saw a CI failure in my GitHub Action from this

```
# Running tests for  runtime...
bin/kubectl wait --for=condition=ready pod runtime-pod --timeout=300s
pod/runtime-pod condition met
bash: -c: line 17: unexpected EOF while looking for matching `"'
```

https://github.com/jiridanek/notebooks/actions/runs/11996005933/job/33439990224#step:29:47

I'll play with it a little to see if I can quickly fix it.

Reply:
done, I've added a commit that removes one space at the end of line to fix this

also removed one extra tab that was spoiling our (attempts at) aesthetically pleasing and easy to navigate indentation